src/components/PopupWithForm.js

```diff
@@ -25,7 +25,6 @@ class PopupWithForm extends Popup {
 
   _setEventListeners() {
```

Comment:
Сlose modal in the case of successful api response in `then` block in `index.js`, otherwise the form will close even if an error occurred and you will need to reopen the form and fill the values again.